src/test/java/pages/AccountModel.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.Select;
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Random;
+
+public class AccountModel {
+    private final WebDriver driver = WebDriverSingletion.getInstance().openBrowser();
+
+    private static final By FIRSTNAME_INPUT = By.name("customer_firstname");
+    private static final By LASTNAME_INPUT = By.name("customer_lastname");
+    private static final By PASSWD_INPUT = By.name("passwd");
+    private static final By ADDRESS_INPUT = By.name("address1");
+    private static final By CITY_INPUT = By.name("city");
+    private static final By POSTCODE_INPUT = By.name("postcode");
+    private static final By STATE_INPUT = By.name("id_state");
+    private static final By PHONE_INPUT = By.name("phone_mobile");
+    private static final By REGISTER_BUTTON = By.cssSelector("button[name=submitAccount]");
+
+    private static final String FIRSTNAME = RandomStringUtils.randomAlphabetic(8);
+    private static final String LASTNAME = RandomStringUtils.randomAlphabetic(8);
+    private static final String PASSWD = getUserRandomPasswd();
```

Comment:
Must be in test class or in some test data factory class, page object is not right place